server/apps/archive/common.py

```diff
@@ -169,16 +169,9 @@ def generate_unique_id_and_name(item, repo_type=ARCHIVE):
     """
 
     try:
-        key_name = 'TEST_{}_SEQ'.format(repo_type.upper()) if superdesk.app.config.get('SUPERDESK_TESTING', False) \
-            else '{}_SEQ'.format(repo_type.upper())
-
-        unique_id = update_key(key_name, flag=True)
-
-        if unique_id:
-            item['unique_id'] = unique_id
-            item['unique_name'] = "#" + str(unique_id)
-        else:
-            raise IdentifierGenerationError()
+        unique_id = str(uuid1())
```

Comment:
Unique Id can't be a string as per the requirement. Please check schema definition https://github.com/superdesk/superdesk-core/blob/master/superdesk/metadata/item.py#L58

Also, as per python docs https://docs.python.org/3.4/library/uuid.html uuid1() generates alpha numeric string..